docs/changelog.rst

```diff
@@ -1,10 +1,28 @@
 Changelog
 ==========
 
+Version 1.0.0
```

Comment:
We're about to release v1.1.1, so maybe we should add a  section for v1.1.0 and v1.1.1